content/_plugins.md

```diff
@@ -412,3 +412,189 @@ Valid values are `tenant` or `global`.
 
 ### Response
 A `Plugin` resource.
+
+## The Plugins Update Resource
+
+### Attributes:
+
+Attribute | Type | Description
+--------- | ------- | -------
+`id` | string | A unique identifier for the plugins update.
+`state` | string | The state of this update (successful, failed, updating, etc...).
+`blueprint_id` | string | The id of the blueprint to its deployments.
+`temp_blueprint_id` | string | The id of the temporary blueprint created for the purpose of this plugins update.
+`execution_id` | string | The id of the execution performing the update.
+`deployments_to_update` | list | A list of deployment updates IDs that are updated in this plugins update.
+`forced` | bool | Whether or not this plugins update is forced.
```

Comment:
Better to explain what is "force" in this context

Reply:
> Whether or not this plugins update was executed regardless of other active/failed plugins updates.
Better?

Reply:
Specifies whether to force the plugin update regardless of other active/failed plugins updates state.